### PR TITLE
Add bio for githubSherryCT.md

### LINKED ISSUE #4: Add bio for githubstudent
- [x] Create a branch
- [x] Add a bio file for githubstudent
- [x] Open a pull request
- [x] Give & receive feedback
- [x] Merge pull request

### PR DESCRIPTION
**Pull request by SherryCT**
This pull request fixes #4 by creating a new bio file for GitHub Student SCT

### CODE REVIEW

File: githubSherryCT.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# GitHub Student
```
Comment (mehrenberg) [inline githubSherryCT.md]: should this be your username?

File: githubSherryCT.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+
+Sherry
```
Comment (alec-at-work) [inline githubSherryCT.md]: hi Sherry!

File: githubSherryCT.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
...
+San Francisco, CA
```
Comment (mehrenberg) [inline githubSherryCT.md]: what's the weather like today?

**Comment by SherryCT** [comment]
:+1:  :100:

**Pull request merged**